server/routes/api/user.js

```diff
@@ -0,0 +1,15 @@
+
+function handle(req, res, database) {
+
+  // get all results from this query
+  database.all("SELECT * FROM Artist", function (err, results) {
+    if (err == undefined) {
+      res.json(results);
```

Comment:
You can see that in `routes/root.js` I used `res.send` to send a string that should be rendered as an HTML page.
To send an object that should be treated as a JSON we use `res.json`.